webcompat/templates/user-activity.html

```diff
@@ -1,23 +1,14 @@
 {% extends "layout.html" %}
 {% block title %}{{user}}'s Activity{% endblock %}
 {% block body %}
-{% include "shared/nav.html" %}
+{% include "nav/nav.html" %}
 <main role="main" data-repo-path="{{config['ISSUES_REPO_URI']}}">
-  <div class="wc-Loader js-loader"></div>
-  <section class="wc-UISection">
-    <div class="wc-UIContent">
-      <h1 class="wc-Title wc-Title--l wc-ResetOffsetBottom js-username">
-        {{user}}'s activity
-      </h1>
-    </div>
+ <!--  <div class="wc-Loader js-loader"></div> -->
```

Comment:
We want to remove this (or file a bug to do some loader action on this page).

Reply:
#1529 -> needs to be added